common/src/main/res/values/strings.xml

```diff
@@ -1274,4 +1275,10 @@
     <string name="state_update_available">Update available</string>
     <string name="state_up_to_date">Up-to-date</string>
     <string name="sensor_name_health_connect">Health Connect sensors</string>
+    <string name="graph_state_widgets">Graph historic widget</string>
+    <string name="widget_graph_we_are_retrieving_data">We are collecting some data…</string>
+    <string name="graph_widget_desc">To see historical data on graph</string>
+    <string name="interval_sampling">Interval sampling:</string>
+    <string name="historical_graph">Historical Graph</string>
+    <string formatted="true" name="graph_time_range">Graph Range (%1$d hours)</string>
```

Comment:
We follow [Material guidelines and use sentence case](https://m3.material.io/foundations/content-design/style-guide/ux-writing-best-practices#fc5c2a78-f4bf-4d42-bdac-42ff80391129) so the second words Graph/Range should be graph/range